tests/unit/utils/Utils.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTILS_H__
+#define __UTILS_H__
+
+/*
```

Comment:
I've added the BSD headers. Not sure if they are the right ones though. Tell me if you prefer a different license version.